packages/react-app/src/lib/bridge.js

```diff
@@ -324,3 +349,16 @@ export const relayTokens = async (ethersProvider, token, receiver, amount) => {
     }
   }
 };
+
+export const wrapAndRelayTokens = async (
```

Comment:
I think it would make more sense to move this to the same method above? and do a switch case for mode === "NATIVE" ?